test/Tests/src/Data/Statistics_Spec.enso

```diff
@@ -0,0 +1,132 @@
+            missing_set.compute Standard_Deviation . should_equal 20.76106661 epsilon=double_error
+            with_nans_set.compute Standard_Deviation . should_equal 20.76106661 epsilon=double_error
+            [1].compute Standard_Deviation . is_nan . should_equal True
+
+        Test.specify "Should be able to get sample standard deviation of values" <|
+            number_set.compute (Standard_Deviation True) . should_equal 23.51410415 epsilon=double_error
+            missing_set.compute (Standard_Deviation True) . should_equal 20.1018163 epsilon=double_error
+            with_nans_set.compute (Standard_Deviation True) . should_equal 20.1018163 epsilon=double_error
+
+        Test.specify "Should be able to get sample skewness of values" <|
+            number_set.compute Skew . should_equal 0.165086552 epsilon=double_error
+            missing_set.compute Skew . should_equal 0.084238123 epsilon=double_error
+            with_nans_set.compute Skew . should_equal 0.084238123 epsilon=double_error
+            [1, 2].compute Skew . is_nan . should_equal True
+
+        Test.specify "Should be able to get population skewness of values" <|
+            number_set.compute (Skew True) . should_equal 0.152437706 epsilon=double_error
+            missing_set.compute (Skew True) . should_equal 0.076125664 epsilon=double_error
+            with_nans_set.compute (Skew True) . should_equal 0.076125664 epsilon=double_error
+            [1, 2].compute (Skew True) . is_nan . should_equal True
+
+        Test.specify "Should be able to get sample kurtosis of values" <|
+            number_set.compute Kurtosis . should_equal -0.449422438 epsilon=double_error
+            missing_set.compute Kurtosis . should_equal -0.201991074 epsilon=double_error
+            with_nans_set.compute Kurtosis . should_equal -0.201991074 epsilon=double_error
+            [1, 2, 3].compute Kurtosis . is_nan . should_equal True
+
+        Test.specify "Should allow bulk computation" <|
+            stats = [Count, Minimum, Mean, Variance, Skew]
+            expected = [20, -45.84, -5.064, 582.0137832, 0.165086552]
+            values = number_set.compute_bulk stats
+            values.map_with_index i->v->((expected.at i - v).abs < double_error) . any v->(v == True) . should_equal True
+
+        Test.specify "Should be able to count valid values on empty Vector" <|
+            [].compute . should_equal 0
+
+        Test.specify "Should fail with Empty_Error on other statistics for empty Vector" <|
+            [].compute Minimum . should_fail_with Vector.Empty_Error
+            [].compute Maximum . should_fail_with Vector.Empty_Error
+            [].compute Sum . should_fail_with Vector.Empty_Error
+            [].compute Mean . should_fail_with Vector.Empty_Error
+            [].compute Variance . should_fail_with Vector.Empty_Error
+            [].compute Skew . should_fail_with Vector.Empty_Error
+            [].compute Kurtosis . should_fail_with Vector.Empty_Error
```

Comment:
It is a bit weird that having too little data for Variance for example, results in NaN if we have length == 1 or 2, but results in empty error if we have length == 0. Seems a bit inconsistent.

Also, since count of empty is just `0`, why not make sum of empty also `0`? It is rather standard that a sum of empty set is defined to be 0.

Reply:
will change all by Min Max to not be an Error as yes no reason to be inconsistent